File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 1 - C/q5.c
#include<stdio.h>
int main(){
	int horaAula, aulasDadas;
	float perc_desc, inss, salario_bruto, salario_liquido;
	printf("Hora Aula: ");
	scanf("%d", &horaAula);
	printf("Aulas Dadas: ");
	scanf("%d", &aulasDadas);
	printf("Percentual de desconto: ");
	scanf("%f", &perc_desc);
	salario_bruto=horaAula*aulasDadas;
	inss=salario_bruto-(salario_bruto*perc_desc)/100;
	salario_liquido=salario_bruto-inss;
	printf("Salario bruto: R$%f\n", salario_bruto);
	printf("Valor do desconto: R$%f\n", inss);
	printf("Salario liquido: R$%f\n", salario_liquido);
}

File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 1 - Python/q5.py
horaAula=int(input("HoraAula: "))
numeroAula=int(input("AulasDadas: "))
porc_desc=float(input("Insira o porcentual de desconto: "))
salario_bruto=horaAula*numeroAula
inss=salario_bruto-(salario_bruto*porc_desc)/100
salario_liquido=salario_bruto-inss

print("Salario bruto: R$", salario_bruto)
print("Valor do desconto: R$", inss)
print("Salario liquido: R$", salario_liquido)
